identity-account-storage/src/types/encrypted_data.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::Deserialize;
+use serde::Serialize;
+
+/// The structure returned after encrypting data
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct EncryptedData {
+  nonce: Vec<u8>,
+  associated_data: Vec<u8>,
+  tag: Vec<u8>,
+  cypher_text: Vec<u8>,
```

Comment:
Nit: should be one word and I think `ciphertext` might be more common? One word either way.

```suggestion
  ciphertext: Vec<u8>,
```